docker/cmd/run.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
```

Comment:
Thanks! Since the shebang is added, could you also `chmod +x run.sh`?

Reply:
added

Reply:
also might want to consider declaring BOOTSTRAP_IP separately before assignment?

Reply:
Will that have specific advantage? Sorry I'm not so familiar with bash script so I failed to see the reason. I do think we might not need `export`

Reply:
probably not for such simple code, but if you ever plan on expanding it, declaring and assigning separately will prevent masking return values in the event of conditionals, etc. 

export won't be really be needed since its not really called externally from the script and is defined in `bootstrap.go` (i'm pretty sure, but may be wrong).

Reply:
hmm does the declaring here mean `set` or `declare` in bash? Anyway, please feel free to modify them and make a PR for it!